versions/3.1.0.md

```diff
@@ -96,6 +96,12 @@ Some examples of possible media type definitions:
 The HTTP Status Codes are used to indicate the status of the executed operation. 
 The available status codes are defined by [RFC7231](https://tools.ietf.org/html/rfc7231#section-6) and registered status codes are listed in the [IANA Status Code Registry](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
 
+##### <a name="summary"></a>Summaries
+In various places throughout the OpenAPI Specification, fields may have a `summary`. The summary is intended to be a short overview of that item and SHOULD be less than 120 characters. It MUST be plaintext and not contain formatting like Markdown or HTML.
```

Comment:
```suggestion
In various places throughout the OpenAPI Specification, objects may have a `summary`. The summary is intended to be a short overview of that item. It MUST be plaintext and not contain formatting like Markdown or HTML.
```

Reply:
Thanks for that background! I have updated the text.